src/main/resources/application.yml

```
@@ -41,3 +41,6 @@ external:
     service-key: ${SERVICE_KEY}
     base-url: https://apis.data.go.kr/1360000/VilageFcstInfoService_2.0
     num-of-rows: 100
+  kakao:
+    rest-api-key: ${REST_API_KEY}
+    base-url: https://dapi.kakao.com/v2/local/geo/coord2regioncode.json
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)